components/widgets/admin_console/admin_panel.tsx

```diff
@@ -37,13 +37,13 @@ const AdminPanel: React.FC<Props> = (props: Props) => (
                         defaultMessage={props.titleDefault}
                     />
                 </h3>
-                <span>
+                <div className='margin-top'>
```

Comment:
Maybe we could change to a name that is more idiomatic that doesn't describe the styling.

Reply:
We should most definitely, bootstrap 4 uses `mt-x` etc, we can use that.
But that can be a separate ticket since these classes are used elsewhere aswell.